sdk/storage/azure-storage-files-datalake/test/ut/datalake_file_client_test.cpp

```diff
@@ -609,6 +609,18 @@ namespace Azure { namespace Storage { namespace Test {
     EXPECT_THROW(ReadFile(testName), std::runtime_error);
   }
 
+  TEST_F(DataLakeFileClientTest, FileDownloadOwnerGroupPermissions)
+  {
+    auto downloadResult = m_fileClient->Download().Value;
+    ASSERT_TRUE(
+        downloadResult.Details.Owner.HasValue() && !downloadResult.Details.Owner.Value().empty());
```

Comment:
Nit: Try not to do more than one check in single test assertions. If this assertion fails, we won't be able to know which check fails.